### ISSUE TITLE
readme correction

### ISSUE THREAD

**Issue by Righolt** [DrGo/sci-write#1]
`(only once per library; already done by SM).` in the setup. The ref keys seem to be local and need to be regenerated for each machine. (Not sure whether this could lead to potential conflicts down the line)

**Comment by DrGo** [comment]
an issue only if you do not set the format of the key (as explained in the README)

**Comment by Righolt** [comment]
The conflict would be if key generation would not be the same in case same author-year lead to different letters (especially if one had a,b etc. pre-existing and an other machine runs from scratch).

**Comment by DrGo** [comment]
the rule should be that only the machine that adds the ref updates the key. keys of existing refs are not updated unless a user choose to.

**Comment by Righolt** [comment]
In a new set-up you don't have any keys though and you need to run. With the current settings the adding machine will gen first, but those keys seems local, because you need to regen when setting it up on a new machine (it even does generates automatically after installing better bibtex).

**Comment by Righolt** [comment]
According to <https://retorque.re/zotero-better-bibtex/citing/#configurable-citekey-generator> a post fix is generated, but there is no information on handling of ties. If date added is the first tiebreaker, this would solve all issues.
